entrypoint.sh

```diff
@@ -2,7 +2,14 @@
 
 rules_repository="$1"
 output_type="$2"
+fail_on_result="$3"
+
+if [[ $fail_on_result == "true" ]]; then
+  fail_on_result_cmd="--fail-on-result";
+else
+  fail_on_result_cmd="--no-fail-on-result";
+fi
 
 cd /github/workspace/
 clj-holmes fetch-rules -r "$rules_repository"
-clj-holmes scan -p . -t "$output_type" -o clj-holmes.sarif
+clj-holmes scan -p . $fail_on_result_cmd -t "$output_type" -o clj-holmes.sarif
```

Comment:
since now it supports specify the output type what do u think about adding an option to specify the filename?

Reply:
Also if the output-type is stdout it does not need the output-file option

Reply:
:+1: